### PR TITLE
NO-ISSUE: feat: Add container build timestamp to the UI

### PR DESCRIPTION
**Pull request by ibazulic**
The UI now shows build timestamps for all container images. If an image is a manifest list, the UI will show the last modified timestamp instead (as manifest lists are created and modified at the same time), while container images might have different build dates.

Co-authored-by: Claude <noreply@anthropic.com>

## Release Notes

* **New Features**
  * Display "Image Built" timestamps for tags and container images, showing when container images were created.
  * Enrich manifest list children with creation timestamps and platform/architecture information (Linux/amd64, Linux/arm64, etc.).
  * Add "Image Built" column in tags table and tag details view.

* **Bug Fixes**
  * Improve OCI manifest validation to correctly distinguish image manifests from artifacts.
  * Standardize placeholder text formatting across the UI.

* **Tests**
  * Add comprehensive test coverage for image built timestamp extraction and display across single images, multi-arch manifests, and edge cases.

### CODE REVIEW

**Comment by ibazulic** [comment]
UI screenshot:
image

File: endpoints/api/test/test_manifest.py [hunk 1]
```diff
@@ -245,3 +257,359 @@ def test_repository_manifest_pull_statistics_multiple_pulls(app, initialized_db)
...
+    # Define builders
+    builder_amd64_docker = DockerSchema2ManifestBuilder()
+    builder_arm64_docker = DockerSchema2ManifestBuilder()
+    builder_amd64_oci = OCIManifestBuilder()
+    builder_arm64_oci = OCIManifestBuilder()
+
+    # First we'll build a Docker v2 schema 2 image
+    # for AMD64
+    builder_amd64_docker.set_config_digest(config_amd64_digest, len(config_amd64_bytes))
+    builder_amd64_docker.add_layer(layer_digest, len(layer_digest))
```
[the remark this answers is not in the document]
Comment (ibazulic) [reply endpoints/api/test/test_manifest.py]: Fixed in new commit.

File: web/playwright/e2e/tags/image-built-timestamp.spec.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      const imageBuiltCell = authenticatedPage
+        .getByRole('row')
+        .filter({hasText: 'latest'})
+        .getByRole('cell', {name: /Image Built/});
+
+      await expect(imageBuiltCell).toBeVisible();
+
+      // Check that it's not n/a (pushed images from Docker should have a timestamp)
+      const cellText = await imageBuiltCell.textContent();
+      expect(cellText).not.toBe('N/A');
```
[the remark this answers is not in the document]
Comment (ibazulic) [reply web/playwright/e2e/tags/image-built-timestamp.spec.ts]: Reworked tests to be better aligned with Playwright guidelines.

File: web/playwright/e2e/tags/image-built-timestamp.spec.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      } catch {
+        // ignore cleanup errors
+      }
+    });
+
+    test('displays image built timestamp column', async ({
+      authenticatedPage,
+    }) => {
+      // Load the page
+      await authenticatedPage.goto(`/repository/${testRepo.fullName}?tab=tags`);
```
[the remark this answers is not in the document]
Comment (ibazulic) [reply web/playwright/e2e/tags/image-built-timestamp.spec.ts:47]: The test follows the same principles as other tests. Guard rails are already applied in fixtures.

**Comment by ibazulic** [comment]
@dmesser You were also working on this issue, so I'd really like your opinion on this as well.

File: web/src/resources/TagResource.ts [hunk 4]
```diff
@@ -53,6 +55,8 @@ export interface Manifest {
   layers: Layer[];
   // Whether manifest content is available locally (not sparse)
   is_present?: boolean;
+  // Read child manifest built timestamp if applicable
+  image_built?: string;
```
[the remark this answers is not in the document]
Comment (ibazulic) [reply web/src/resources/TagResource.ts:78]: Should be resolved in the new commit.

**Comment by ibazulic** [comment]
New screenshot:

image
